settings.gradle.kts

```diff
@@ -60,3 +53,42 @@ gradleEnterprise {
         publishAlwaysIf(isCiBuild)
     }
 }
+
+@Suppress("UnstableApiUsage") // Central declaration of repositories is an incubating feature
```

Comment:
I believe no need to suppress these `UnstableApiUsage`s.

Reply:
it's still incubating https://docs.gradle.org/7.6/userguide/declaring_repositories.html#sub:centralized-repository-declaration

> Central declaration of repositories is an incubating feature



without it, there are several warnings

<img width="1080" alt="image" src="https://user-images.githubusercontent.com/897017/215991074-d3669f05-cb81-432f-8414-064e35d6b85e.png">

Reply:
There might be more places that need to be suppressed, personally think it's unnecessary.